src/loaders/FileLoader.js

```diff
@@ -75,7 +75,7 @@ class FileLoader extends Loader {
 
 		// create request
 		const req = new Request( url, {
-			headers: new Headers( this.requestHeader ),
+			headers: this.requestHeader,
```

Comment:
It think it's better for consistency to use the `Headers` interface even that means you have to use lower case letters. Especially with regard to https://github.com/fastify/help/issues/71#issuecomment-482136803.

> Also, the Internet will eventually move to HTTP/2, which only supports lowercase headers, so it's not worth getting attached to the aesthetic look of HTTP/1-style headers.